types/stylelint/index.d.ts

```diff
@@ -364,6 +364,77 @@ declare module 'stylelint' {
 			line?: number;
 		};
 
+		export type KeywordSets = {
```

Comment:
[suggest] Could you add a type annotation to each module like this? By doing so, we could improve the code's type safety.

```js
// lib/reference/keywordSets.js

/** @type {import('stylelint').KeywordSets} */
const keywordSets = {};

// ...

module.exports = (keywordSets);
```
```